staging/src/k8s.io/apiserver/pkg/util/webhook/webhook.go

```diff
@@ -32,13 +32,21 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 )
 
+// defaultRequestTimeout is set for all webhook request. This is the absolute
+// timeout of the HTTP request, including reading the response body.
+const defaultRequestTimeout = 20 * time.Second
```

Comment:
nit: For consistency with the [admission webhook](https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/admission/webhook/admission.go#L284), could you set it to 30s?

Reply:
done